CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Version History
 
+- v4.2.3 [#683](https://github.com/davidjbradshaw/iframe-resizer/issues/683) Include border top/bottom, plus padding top/bottom, when calculating heights on iframe with `box-sizing: border-box;`
```

Comment:
Don’t forget to add a credit link I g back to your GitHub account :)

Reply:
Thanks - wasn't sure how links worked at first, but I get it now